app/models/kuroko2/job_definition.rb

```diff
@@ -81,7 +81,7 @@ def proceed_multi_instance?
   end
 
   def text_tags
-    tags.pluck(:name).join(',')
+    tags.map(&:name).join(',')
```

Comment:
Why this change needed?

Reply:
Without this change, `text_tags` of duplicated definition always returns empty.

```ruby
definition = Kuroko2::JobDefinition.first
duplicated_definition = definition.dup
duplicated_definition.tags = definition.tags.dup

definition.tags.pluck(:name) #=> ["HelloWorld"]
definition.tags.map(&:name) #=> ["HelloWorld"]
duplicated_definition.tags.pluck(:name) #=> []
duplicated_definition.tags.map(&:name) #=> ["HelloWorld"]
```

Reply:
`text_tags` method is called at [here](https://github.com/cookpad/kuroko2/blob/52c077269f7613546f88152e00e02c3862e50098/app/views/kuroko2/job_definitions/_form.html.slim#L85).
In [new page](https://github.com/cookpad/kuroko2/blob/52c077269f7613546f88152e00e02c3862e50098/app/views/kuroko2/job_definitions/new.html.slim),  `text_tags` would return  empty array without this change because the job_definition having this `tags` does not exists in DB (so `#pluck` returns empty array).   

Is it right? @takonomura

Reply:
Yes